extensions/spring-scheduling/deployment/src/main/java/io/quarkus/spring/scheduling/deployment/SpringSchedulingProcessor.java

```diff
@@ -0,0 +1,97 @@
+package io.quarkus.spring.scheduling.deployment;
+
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+import org.jboss.jandex.AnnotationInstance;
+import org.jboss.jandex.AnnotationTarget;
+import org.jboss.jandex.AnnotationValue;
+import org.jboss.jandex.DotName;
+import org.jboss.jandex.MethodInfo;
+import org.jboss.logging.Logger;
+
+import io.quarkus.arc.deployment.AdditionalBeanBuildItem;
+import io.quarkus.arc.deployment.AnnotationsTransformerBuildItem;
+import io.quarkus.arc.processor.AnnotationsTransformer;
+import io.quarkus.arc.processor.DotNames;
+import io.quarkus.deployment.annotations.BuildProducer;
+import io.quarkus.deployment.annotations.BuildStep;
+import io.quarkus.deployment.builditem.CombinedIndexBuildItem;
+import io.quarkus.deployment.builditem.FeatureBuildItem;
+
+/*
+ * A simple processor that maps annotations Spring DI annotation to CDI annotation
+ * Arc's handling of annotation mapping (by creating an extra abstraction layer on top of the Jandex index)
+ * suits this sort of handling perfectly
+ */
+public class SpringSchedulingProcessor {
+
+    static final DotName SPRING_SCHEDULED = DotName.createSimple("org.springframework.scheduling.annotation.Scheduled");
+
+    private static final DotName QUARKUS_SCHEDULED = DotName.createSimple(io.quarkus.scheduler.Scheduled.class.getName());
+    private static final Logger LOGGER = Logger.getLogger(SpringSchedulingProcessor.class);
+
+    @BuildStep
+    FeatureBuildItem registerFeature() {
+        return new FeatureBuildItem(FeatureBuildItem.SPRING_SCHEDULING);
+    }
+
+    @BuildStep
+    AnnotationsTransformerBuildItem buildScheduledAnnotationTransformer() {
+        return new AnnotationsTransformerBuildItem(new AnnotationsTransformer() {
+            @Override
+            public boolean appliesTo(AnnotationTarget.Kind kind) {
+                return kind == AnnotationTarget.Kind.METHOD;
+            }
+
+            @Override
+            public void transform(TransformationContext context) {
+                MethodInfo methodInfo = context.getTarget().asMethod();
+                List<AnnotationInstance> collect = methodInfo.annotations().stream()
+                        .filter(annotationInstance -> annotationInstance.name().equals(SPRING_SCHEDULED))
```

Comment:
Is this annotation repeatable? If so, there's a good chance the repeatable container will need some special handling (and a test).

Reply:
I don't get what do you mean. Yes, this annotation is repeatable.

Reply:
@gsmet means that you will also have to handle `@Schedules`